idaes/core/surrogate/pysmo/tests/test_kriging.py

```diff
@@ -97,6 +97,7 @@ def test__init__07(self, array_type):
 
     @pytest.mark.unit
     @pytest.mark.parametrize("array_type", [np.array, pd.DataFrame])
+    @pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
Do you have a list of the `RuntimeWarning`s that are being filtered here?

Reply:
I am filtering two types of runtime warnings:

`RuntimeWarning: invalid value encountered in log
RuntimeWarning: divide by zero encountered in log`
  
However, I'm happy to remove the filters - the runtime warnings are few (about 10 of them). I'm on the fence about whether to filter them or not.

Reply:
I see, thanks. If I understand how `pytest.mark.filterwarnings` works, I think you should be able to use a regular expression to catch both of those cases:

```suggestion
    @pytest.mark.filterwarnings("ignore:(?:invalid value|divide by zero) encountered in log:RuntimeWarning")
```

The goal that we're aiming to is to have no warnings at all that are not expected, so `pytest.mark.filterwarnings` is a good approach, as long as the filter is as specific is possible. In this case, for example, not adding a regex would let possible future `RuntimeWarning` being raised go undetected (which we don't want).

Reply:
Oh, I didn't know I could do that. 

I agree on the "specific" bit - that is why I was on the fence about adding the filters to start with. 

I'll make the change and see it it works. 

Thanks!

Reply:
Resolved